### ISSUE TITLE
Stop House Robberies Job giving you your own property/house

### ISSUE THREAD

**Issue by MrsTORN** [echoroleplay/feedback#124]
**Suggestion Description**
Link Up House robberies Job doesn't give you a 'house robbery' for your own property.

**Reason**
my reason is because I queued up for a house and the house it gave me to rob was my own property, I think i've heard others having the same issues.

**Comment by davidxd33** [comment]
I don't think this should be implemented at all. It shouldn't matter if the location it gives you is a player's house or not even if in this particular case the location ended up being a house you own by coincidence. 

If anything, it should further enhance the situation if you know you are robbing the location of a house you know belongs to someone through RP.

**Comment by damightylamb** [comment]
@davidxd33 it does matter. You can't rob your own house because it's failrp and kinda LQ. Plus, admins already said you can't rob it.

**Comment by davidxd33** [comment]
I suppose about the "your own house" part that shouldn't happen. Checks could be put in place to give another location

**Comment by FlawwsX** [comment]
Not sure how the checks would work. I do plan to implement a "re-roll" option for the location.
